### PR TITLE
Tweak sizes and sizePolicies to stop layout shifting

### PR DESCRIPTION
**Pull request by smithrobs**
Attempting to ameliorate #75 a bit as well as tweak the sizes of things. Looks better on my MBP but needs to be tested on Win/Linux before merging.

### CODE REVIEW

File: zoia_lib/UI/ZOIALibrarian.ui [hunk 1]
```diff
@@ -435,7 +435,16 @@
                 <number>10</number>
                </property>
                <widget class="QWidget" name="layoutWidget">
+                <property name="sizePolicy">
```
Comment (smithrobs) [inline zoia_lib/UI/ZOIALibrarian.ui:438]: These sizePolicy and sizeConstraint seem to be what made the most difference in terms of stopping the layout recalc/shift

**Comment by meanmedianmoge** [comment]
@smithrobs still seeing the snap - I upgraded to PySide 5.15.1 as well

**Comment by smithrobs** [comment]
I was worried this was a "works on my machine (but nobody else's)" situation. Alright, I'll keep messing with the layout.

**Comment by meanmedianmoge** [comment]
Ps @smithrobs I merged this into the v1.1 branch. Not quite fixing the snap yet, but I just wanted to mention it.

**Pull request closed without merging**